src/addrman.cpp

```diff
@@ -759,8 +759,9 @@ void AddrManImpl::GetAddr_(std::vector<CAddress>& vAddr, size_t max_addresses, s
 
     // gather a list of random nodes, skipping those of low quality
     const int64_t now{GetAdjustedTime()};
+    std::vector<CAddress> addresses;
```

Comment:
Tiny nit for one of the refactor PRs: `addresses.reserve(std::min(vRandom.size(), nNodes));`